src/soundmanager.h

```diff
@@ -62,13 +62,13 @@ class SoundManager : public QObject {
     // open, this method simply runs through the list of all known soundcards
     // (from PortAudio) and attempts to close them all. Closing a soundcard that
     // isn't open is safe.
-    void closeDevices();
+    void closeDevices(bool wait);
```

Comment:
I would like to get away from bare booleans as arguments.  They are hard to understand at the call site without explanatory comments ("closeDevices(/*wait*/ false);")

http://jlebar.com/2011/12/16/Boolean_parameters_to_API_functions_considered_harmful..html

For this "wait" parameter, perhaps a time argument is appropriate, and then the "5" doesn't have to be hardcoded.  A caller can pass 0 for no waiting.